ansible/playbooks/adhoc/hotfix/roles/update_master/tasks/main.yml

```diff
@@ -36,6 +34,7 @@
 - name: Set downgrade_successful=True if downgrade succeeded
   set_fact: 
     downgrade_successful: True
+    upgrade_successful: True
```

Comment:
Thanks, I like that a lot better. It helps avoid bugs like the previous one where I had the names confused.